content/blog/foreign-key-constraints.md

```diff
@@ -0,0 +1,198 @@
+---
+author: "Rob Ashton"
+date: 2019-06-19
+title: Querying for foreign key constraints
+best: false
+---
+
+Over the past few months we've been developing a tool to assist with database import from R. As part of that we want to automate some of the import steps to both reduce the development work needed for each import and to make code review easier. We want to be able to take a list of data frames in R and append those to the relevant tables in the database automatically. Doing so requires knowledge of the foreign key constraints, this blog post covers how to query PostgreSQL and SQLite databases to retrieve that information. 
+
```

Comment:
Grammar pedantry: Edit the comma in the last sentence - I think a ';' will do, or split into two sentences.